contracts/cw-proposal-single/src/error.rs

```diff
@@ -36,9 +36,12 @@ pub enum ContractError {
     #[error("Not registered to vote (no voting power) at time of proposal creation.")]
     NotRegistered {},
 
-    #[error("Already voted")]
+    #[error("Already voted. This proposal does not support revoting.")]
     AlreadyVoted {},
 
+    #[error("Already cast that vote. Change your vote to revote.")]
```

Comment:
This isn't the most clear as we use AlreadyVoted above, "Already cast a vote with this option."

Reply:
Great point. Updated the wording.